src/components/todo/List.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'lodash';
+import React, { Component } from 'react';
+import TodoInput from './Input.js';
+import TodoEntry from './Entry.js';
+
+class TodoList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      entries: JSON.parse(localStorage.getItem('entries')) || []
```

Comment:
Will be better to extract such things to a singleton. And separate view and global data manipulation from each other.